client/test/cypress/support/index.js

```diff
@@ -0,0 +1,27 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands';
+
+const resizeObserverLoopError = 'ResizeObserver loop limit exceeded';
+
+Cypress.on('uncaught:exception', (err) => {
+  if (err.message.includes(resizeObserverLoopError)) {
```

Comment:
Would you mind helping us understanding this part? After doing some googling I would assume that the return value should be true here.

Reply:
That's boilerplate from adding cypress. I'll dig in a figure out what it's doing.